distributions/aws/examples/storage-efs/README.md

```diff
@@ -0,0 +1,153 @@
+# Deploying Kubeflow with AWS EFS as Persistent Storage
+
+This guide describes how to use Amazon EFS as Persistent storage with Kubeflow.  
+
+## 1.0 Prerequisites
+1. For this README, we will assume that you already have an EKS Cluster with Kubeflow installed since the EFS CSI Driver can be installed and configured as a separate resource on top of an existing Kubeflow deployment. You can follow any of the other guides to complete these steps - choose one of the AWS managed service integrated offering<link> or generic distribution<link>.
```

Comment:
Probably doesn't need the `1.` here @mbaijal

Reply:
why don't we link existing README on how to install kubeflow ?

Reply:
I have placeholders for them `<link>`. The upstream directory structure is still called examples so I did not specify the path yet.

Reply:
done.